src/irs/core/storage_inspector.py

```diff
@@ -55,6 +55,62 @@ def __init__(
         self._staging_object_dao = staging_object_dao
         self._object_storages = object_storages
 
+    async def _inspect_object(
+        self, *, bucket_id: str, object_id: str, storage_alias: str
+    ):
+        """Check one specific object in a specified storage node."""
+        try:
```

Comment:
```suggestion
        """Check one specific object in a specified storage node."""
        extra = {
            "object_id": object_id,
            "bucket_id": bucket_id,
            "storage_alias": storage_alias,
        }
        
        try:
```
Then you could add the file_id in the case of a stale object